src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/91411692/41e836bb-a3a0-4b13-9a03-4363f4534ca3)

add types to the PR